Twig/Extension/SonataAdminExtension.php

```diff
@@ -171,6 +174,8 @@ public function renderListElement(
     }
 
     /**
+     * @internal
```

Comment:
> Could we instead try to change the test to be able to make this private?

I'll try to have a look at that, maybe it can be done with `ReflectionMethod::setAccessible()`